### PR TITLE
ID edits- first batch

### PR DESCRIPTION
**Pull request by dmuelle**
Edits for first few sections- up until the **Developing a RESTful microservice** section, which will be covered in the next PR

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -21,28 +21,28 @@
 :page-seo-title: Implementing a microservice using Jakarta EE and MicroProfile API
 :page-seo-description: A tutorial with examples on how to implement a microservice using Jakarta EE and Eclipse MicroProfile API.
 :guide-author: Open Liberty
-= A Technical Deep on Liberty
+= A Technical review of Liberty
```
Comment (dmuelle) [inline README.adoc]: I think this was intended to be "Deep Dive"? IBM terminology advises against this term:

deep dive  (noun)
    Do not use. Do not use to describe a training class, a conference session, a particularly thorough meeting, or a detailed explanation, unless its use is already established for a product. Depending on the context, use wording such as "in-depth analysis" or "technical review."
    
I'm not sure if "Technical Review" is the best title for this though. I think something like one of the following could also work
- Liberty: An in-depth tutorial
- A  technical tutorial on Liberty
- The Liberty tutorial
- An in-depth guide to Liberty

File: README.adoc [hunk 2]
```diff
@@ -210,7 +210,7 @@ mvn liberty:stop
 
 The Liberty Maven plug-in includes a `dev` goal that listens for any changes in the project, including application source code or configuration. The Liberty server automatically reloads the configuration without restarting. This goal allows for quicker turnarounds and an improved developer experience.
 
-Stop the Liberty server if it is running, and start it in dev mode by running the `liberty:dev` goal in the `start/inventory` directory:
+If the Liberty server is running, stop it and restart it in dev mode by running the `liberty:dev` goal in the `start/inventory` directory:
```
Comment (dmuelle) [inline README.adoc:213]: end of review for this PR

File: README.adoc (hunk 1 above)
Comment (gkwan-ibm) [reply README.adoc]: I suggest "An in-depth tutorial on Liberty" and asking the team

Comment (gkwan-ibm) [reply README.adoc]: The team told me that "A Technical Deep on Liberty" is the workshop title and cannot be changed

File: README.adoc [hunk 3]
```diff
@@ -111,13 +111,13 @@ endif::[]
...
+Liberty now offers an easier way to get started with developing your application: the Open Liberty Starter. This tool provides a simple and quick way to get the necessary files to start building an application on Liberty. Through this tool, you can specify your application and project name. You can also choose a build tool from either Maven or Gradle, and pick the Java SE, Jakarta EE, and MicroProfile versions for your application.
 
 In this workshop, the Open Liberty Starter is used to create the starting point of the application. Maven is used as the selected build tool and the application uses of Jakarta EE 9.1 and MicroProfile 5.
 
 To get started with this tool, see the Getting Started page: https://openliberty.io/start/[https://openliberty.io/start/^]
 
-When there, enter the properties that are needed for the application.
+Onn that page, enter the following properties in the **Create a starter application** wizard.
```
Comment (gkwan-ibm) [inline README.adoc]: `Onn` -> `On`

File: README.adoc (hunk 1 above)
Comment (dmuelle) [reply README.adoc]: restored the title

File: README.adoc (hunk 3 above)
Comment (dmuelle) [reply README.adoc]: fixed

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**